tests/plugins/ml/models/create_metadata.yaml

```diff
@@ -0,0 +1,82 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
```

Comment:
The API here is `models/meta` so the file should be `meta.yaml`. And since we're not testing the plugin itself but the API it should not be uploading chunks, instead create a `chunk.yaml` in which that is done to test the `chunk` API (in that one `/meta` will be invoked in a prologue).